### DISCUSSION TITLE
laravel 8.17.1 profile-photos in public storage file not getting propagated

### DISCUSSION THREAD

**Discussion by rhoyle** [laravel/framework#35464] [Q&A]
- Laravel Version: 8.17.1
- PHP Version: 7.4.10
- Database Driver & Version:

### Description:
not able to see profile-photos  the photos shows up in the storage/public/profile-photos file and in the database but not in the public/storage/profile-photos file and therefore not in the dashboard header either.

### Steps To Reproduce:
I just did the normal Laravel new project name --jer
registered and went to the profile page to choose a photo the initial photo showed up but after saving it I just got my name. and know photo. when I checked the database the profile_photo_path was there so I went to the storage\app\ and found a livewire-temp file with a image in it.  Next I went to the storage\app\public\profile-photos folder and it to had a image in it. But when I went to public\Storage\Profile-photos it was empty.  so I then ran the php artisan storage:link command but it said that the "link already exists".  
now what? why is it not putting the image in the public folder?  what do I nave to do to get it to do so?  if I copy the image from the storage folder to the public folder it works just fine.  It seems to me that the automatic copy function is not working.

**Comment by rhoyle** [comment]
PS this did work in Laravel 8.12.3 just fine.

**Comment by Ifriqiya** [comment]
I don't know if this matters but check to confirm if the referenced folder names  in your config/filesystems.php file are identical to the actual folder names in your app. Maybe you have public\Storage\Profile-photos instead of public\storage\profile-photos. Capital 'P' and 'S' instead of 'p' and 'S'.
